.github/workflows/ci.yml

```
@@ -27,6 +27,24 @@ jobs:
         python -m pip install --upgrade pip setuptools wheel
         python -m pip install -q --no-cache-dir -e .[complete]
         python -m pip list
+    - name: Unit Tests
+      run: |
+        python -m pip install pytest
+        py.test --junitxml=./reports/junit.xml -o junit_suite_name=pylhe $GITHUB_WORKSPACE
```

```suggestion
        python -m pytest --junitxml=./reports/junit.xml -o junit_suite_name=pylhe $GITHUB_WORKSPACE
```

`pytest` should always be run with `python -m pytest` as we have observed multiple times over the years that `py.test` does not always give the expected runtime behaviour.